docs/mpid_messaging.md

```diff
@@ -32,13 +32,15 @@ This is a simple data structure for now and will be a ```std::map``` ordered by
 ```c++
 struct MpidMessage {
   PublicMpid::Name sender;
-  std::vector<PublicMpid::Name> recipients;
-  std::string message, body;
+  PublicMpid::Name recipient;
+  std::string message_head, message_body;
```

Comment:
If message_head needs to be part of the MpidHeader and will be sent to the receiver directly,
I strongly suggest to have a limited size of head part. so the struct shall be defined as something like : 
std::string message_body;
BoundedString<0, 1024> message_head;